internal/crdReplicator/peeringPhase_test.go

```diff
@@ -205,7 +205,7 @@ var _ = Describe("PeeringPhase-Based Replication", func() {
 
 				controller.AddedHandler(c.resource, gvr)
 
-				_, err := controller.LocalDynClient.Resource(gvr).
+				_, err := controller.LocalDynClient.Resource(gvr).Namespace("default").
```

Comment:
Would use the same constant as the rest of the test, to avoid issues if it were changed.